keylime/policy/create_runtime_policy.py

```diff
@@ -396,7 +396,7 @@ def process_flat_allowlist(allowlist_file: str, hashes_map: Dict[str, List[str]]
                 pieces = line.split(None, 1)
                 if not len(pieces) == 2:
                     logmsg = f"Skipping line that was split into {len(pieces)} parts, expected 2: {line}"
```

Comment:
Not part of this PR, but why are the log messages separately? I assume this is because of the pylint warning, that one should pass variables as positional arguments instead of using format strings and this works around it. Not that it really matters here, but its kinda weird.

Reply:
I don't recall why, to be honest, and looking at it again, I have to agree it looks very odd. I don't think it was to work around pylint, though. We should probably improve that at some point.

Reply:
Thanks!